relay-server/src/actors/spooler/mod.rs

```diff
@@ -457,9 +488,26 @@ impl OnDisk {
         .await
         .map_err(BufferError::InsertFailed)?;
 
+        self.track_count(1);
         relay_statsd::metric!(counter(RelayCounters::BufferWrites) += 1);
         Ok(())
     }
+
+    fn track_count(&mut self, increment: i64) {
+        // Track the number of envelopes read/written:
+        let metric = if increment < 0 {
+            RelayCounters::BufferEnvelopesRead
+        } else {
+            RelayCounters::BufferEnvelopesWritten
+        };
+        relay_statsd::metric!(counter(metric) += increment);
+
+        // If used, also track the total envelope count:
```

Comment:
What does "if used" mean?

Reply:
Will remove the comment, it does not add any information.